packages/hub/services/queries/sent-push-notifications.ts

```diff
@@ -0,0 +1,54 @@
+import DatabaseManager from '@cardstack/db';
+import { inject } from '@cardstack/di';
+import { buildConditions } from '../../utils/queries';
+
+interface SentPushNotificationsIndex {
+  transactionHash: string;
+  ownerAddress: string;
+  pushClientId: string;
+}
+
+interface PushNotificationData extends SentPushNotificationsIndex {
+  notificationType: string;
+  notificationTitle?: string;
+  notificationBody: string;
+  notificationData?: {};
+}
+
+export default class SentPushNotificationsQueries {
+  databaseManager: DatabaseManager = inject('database-manager', { as: 'databaseManager' });
+
+  async insert(model: PushNotificationData) {
+    let db = await this.databaseManager.getClient();
+
+    await db.query(
+      'INSERT INTO sent_push_notifications (transaction_hash, owner_address, push_client_id, notification_type, notification_title, notification_body, notification_data) VALUES($1, $2, $3, $4, $5, $6, $7)',
+      [
+        model.transactionHash,
+        model.ownerAddress,
+        model.pushClientId,
+        model.notificationType,
+        model.notificationTitle,
+        model.notificationBody,
+        model.notificationData,
+      ]
+    );
+  }
+
+  async exists(index: SentPushNotificationsIndex): Promise<any> {
```

Comment:
Should we name this SentPushNotificationsFilter - for consistency with other filtering queries?

Reply:
I was hoping to communicate that this was the unique index of the table and that these fields were necessary to identify a notification for an event. Reading this I realise it is probably better to use a comment to highlight this at the type and use the convention of naming the parameter `filter` for db reads.